doc/UserGuide.md

```diff
@@ -0,0 +1,151 @@
+   1. Set "Memory" to 16.0 GiB or higher.
+   1. Click "Apply & Restart" and wait for the green "Docker Engine is running" message to reappear.
+   
+   <p align="center"><img src="images/advanced_settings.png" width="75%"></p>
+
+1. Download [**apt-insar.sh**](https://asfdaac.s3.amazonaws.com/s1tbx-rtc.sh) to the directory where RTC products should be saved.
+
+</details>
+
+<details><summary>Ubuntu 18.04</summary>
+
+1. Install Docker using apt
+   ```
+   sudo apt update
+   sudo apt install -y docker.io
+   ```
+1. Add your user to the docker group
+   ```
+   sudo usermod -aG docker $USER
+   ```
+1. Log out and log back in for the group change to take effect
+1. To verify everything is working run the docker command
+   ```
+   docker run hello-world
+   ```
+   Confirm you see the following in your output
+   ```
+   Hello from Docker!
+   This message shows that your installation appears to be working correctly.
+   ```
+1. Download **apt-insar.sh** to the directory where RTC products should be saved
+   ```
+   wget https://raw.githubusercontent.com/asfadmin/grfn-s1tbx-rtc/master/scripts/s1tbx-rtc.sh
+   ```
+</details>
+
+<details><summary>CentOS 7</summary>
+
+1. Install Docker
+   ```
+   curl -fsSL https://get.docker.com/ | sh
+   ```
+1. Create a docker group and add your user to it
+   ```
+   sudo groupadd docker
+   sudo usermod -aG docker $USER
+   ```
+1. Log out and log back in for the group change to take effect
+1. Start Docker
+   ```
+   sudo service docker start
+   ```
+1. To verify everything is working run the docker command
+   ```
+   docker run hello-world
+   ```
+   Confirm you see the following in your output
+   ```
+   Hello from Docker!
+   This message shows that your installation appears to be working correctly.
+   ```
+1. Download **apt-insar** to the directory where InSAR products should be saved
+   ```
+   wget https://raw.githubusercontent.com/asfadmin/grfn-s1tbx-rtc/master/scripts/apt-insar.sh
+   ```
+</details>
+
+## Usage
+
+1. Find the name of the GRD or SLC granule to process from [Vertex](https://vertex.daac.asf.alaska.edu/).
+   
+   *The examples below use S1B_IW_GRDH_1SDV_20190430T161529_20190430T161554_016038_01E295_771B and S1B_IW_GRDH_1SDV_20190430T161529_20190430T161554_016038_01E295_771B*.
+
+1. **macOS only**
+
+   1. Open the Terminal app
+
+      <p align="center"><img src="images/terminal.png" width="75%"></p>
+   
+   1. In your Terminal window, navigate to the directory where **apt-insar.sh** is saved.
+   
+      *For example, if you saved the script to your Downloads directory, type:*
+      ```
+      cd ~/Downloads
+      ```
+
+1. Execute **apt-insar.sh** with the granule name and desired options
+   ```
+   sh apt-insar.sh -p S1B_IW_GRDH_1SDV_20190430T161529_20190430T161554_016038_01E295_771B -s S1B_IW_GRDH_1SDV_20190430T161529_20190430T161554_016038_01E295_771B
```

Comment:
`-r`, probably worth using the long names for clarity `--reference-scene` `--secondary-scene`